Doberman/Pipeline.py

```diff
@@ -41,7 +41,7 @@ def create(config, **kwargs):
         Creates a pipeline and returns it
         """
         for node in config['pipeline']:
-            if node['type'] == 'SensorSourceNode':
+            if node['type'] in ['SensorSourceNode', 'DeviceRespondingSyncNode']:
```

Comment:
Just a matter of style but I would add a class attribute to Node: `creates_sync_pipelines = False` and then override it in those that do to avoid the class checking here which seems a bit ugly

Reply:
Declined